app/routes/participants/profile.js

```diff
@@ -2,6 +2,11 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
     model(params) {
-        return this.store.findRecord('profile', params.profile_id);
+        return Ember.RSVP.hash(
```

Comment:
I'm observing that this model hook is not running when using the "View profile" helper generated via the linkto here: https://github.com/CenterForOpenScience/experimenter/pull/14/files#diff-3ba2795ec3903c0cfb79ef850dd1b844R1

Seems like this may be a common/easily resolved issue:
http://discuss.emberjs.com/t/model-hook-does-not-get-called-for-nested-routes/7838